addon-test-support/@ember/test-helpers/settled.js

```diff
@@ -200,5 +201,8 @@ export function isSettled() {
 export default function settled() {
   let options = arguments[0];
 
+  assert(`The 'settled' (formerly 'wait') testing helper only accepts 'undefined' or an object. You passed ${options}`,
+    typeof options === 'undefined' || typeof options === 'object' && options !== null);
```

Comment:
why is `undefined` a valid value? `settled()` and `settled(undefined)` are somewhat similar but I don't think we should encourage passing `undefined` here.

Reply:
Because that’s what the value appears to be for `settled()`. I don’t think _actually_ passing undefined is useful in anyway.

Reply:
yep, that is why I don't think we should mention it that way in the assertion message. also we could check if `arguments.length > 0`, because IMHO `settled(undefined)` should also throw the assertion.

Reply:
Oh I see. Sure, that makes sense.